.github/workflows/export-dynamic.yaml

```diff
@@ -1,31 +1,22 @@
 name: Export Dynamic Plugin Packages
 on:
-  workflow_dispatch:
+  workflow_call:
     inputs:
       node-version:
         description: node-version to execute the export
-        required: false
-        type: choice
+        type: string
+        required: true
         default: '20.x'
-        options:
-          - '18.x'
-          - '20.x'
 
       janus-cli-version:
         description: Version of the janus-idp/cli package.
         type: string
-        required: false
+        required: true
         default: '^1.18.0'
 
-      overlay-branch:
```

Comment:
Why did you remove this one completly ?

Reply:
I was keeping as default only the branch from where the job started, but I will modify and keep the logic to optionally setup an overlay branch.